map2.js

```diff
@@ -31,3 +31,14 @@ var phoneBook = [
 // For example, the 0th element of the new array should be {Abe: "1-111-111-1111"}
 
 //Your code here
+const addCountryCode = (obj) => {
+  const phoneNumber = obj[Object.keys(obj)[0]];
+  const countryCode = phoneNumber.charAt(0);
+  obj[Object.keys(obj)[0]] = countryCode + '-' + phoneNumber;
+  return obj;
+}
+
+const newPhoneBook = phoneBook.map(addCountryCode);
+console.log(newPhoneBook);
+
+//*googled mozilla.developer for object.keys and charAt
```

Comment:
nice googling! another thing to keep in mind is that arrays and strings both have index values. for an array, if you do `array[0]`, it gets you the first element in the array. for a string, doing `string[0]` will return the first **letter** of the string. Since the phone numbers are strings, you can also use `phoneNumber[0]` to get the first character in that string.